Source/Scene/Cesium3DTileset.js

```diff
@@ -523,6 +524,14 @@ define([
         this.loadSiblings = defaultValue(options.loadSiblings, false);
 
         /**
+         * An array of {@link Plane} used to clip the tileset.
```

Comment:
Here and throughout the reference doc should provide a tad more context: what does it mean to "clip" the tileset?